### DISCUSSION TITLE
Prose-form citations with non-breaking space?

### DISCUSSION THREAD

**Discussion by lumpiluk** [typst/typst#4808] [Q&A] [answered]
I'm using the IEEE citation style with a lot of `#cite(form: "prose", …)`, which produces output like "Author et al. [1]". Unfortunately, the space between the author name(s) and "[1]" is a breaking space. Is there some way for me to change this?

**Comment by laurmaedje** [comment, marked as answer]
Try this rule:
```typ
#show cite: it => {
  show " ": [~]
  it
}
```
Note that this will affect all spaces though. 

Affecting only the last space is trickier, in particular because of #86, so you can't just do `show " [": ..`.
